react/components/EditorContainer/StoreEditor/Styles/StyleEditor/typography/FileFontEditor.tsx

```diff
@@ -131,6 +150,10 @@ const FontFileItem: React.FunctionComponent<FontFileItemProps> = ({
     onChange: (_: React.ChangeEvent, value: string) =>
       onStyleUpdate(stringToFlavour(value)),
     options: STYLE_FLAVOUR_OPTIONS,
+    placeholder: intl.formatMessage({
```

Comment:
You can use the value of `pickStylePlaceholder` returned from the `defineMessages` at the beginning of this file!